### PR TITLE
Fix missing task decorator on allocation expiration warning task

### PR DESCRIPTION
**Pull request by Mark-Powers**

### CODE REVIEW

File: allocations/tasks.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def _send_expiration_warning_mail(alloc, today):
 
     return mail_sent
 
-
+@task
```
[the remark this answers is not in the document]
Comment (super-cooper) [reply allocations/tasks.py:81]: Actually, could the task be run at a specific time? The original PR had this decorator:
https://github.com/ChameleonCloud/portal/blob/alloc-expire-warn/allocations/tasks.py#L83

We had specifically set it to only run once per day in the original PR: https://github.com/ChameleonCloud/portal/pull/217

**Review by super-cooper — CHANGES_REQUESTED** [review]
Sorry, 1 thing!

File: allocations/tasks.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (super-cooper) [reply allocations/tasks.py:81]: nvm

**Review by super-cooper — APPROVED** [review]
Whoops!

**Review by zhenz — APPROVED** [review]
:thumbsup:

**Pull request merged**